9-2-Coaching-Expectations.md

```diff
@@ -0,0 +1,24 @@
+We need coaches to hover around and help our participants with any questions or other issues they might have as they go through the course content. There are a few roles our coaches can take on throughout the day:
+
+# Coaching roles
+
+## Slack channel monitors
+
+Some people will feel comfortable asking questions in our Slack channel, so we need a few people ready to answer them.
+
+## Table helpers
+
+Other participants might be more comfortable with in-person assistance, so we have a coach sitting at each table, ready to help.
+
+## Roamers
+
+A couple of people can wander the room, providing ad-hoc assistance where other helpers are already occupied.
```

Comment:
Potentially mention that our coaches should encourage attendees to raise their hand when they need help; or let the coaches know to look for 'furrowed brows'. Our helpers for the Haskell workshop mentioned this and it put us at ease for asking or expecting help.

Reply:
I've added a note about the second bit - re: the first comment, I feel like we want the speakers (either semi-regularly or as part of the rest of their intro for the day) to encourage people to raise their hand if they want help - where do we put that guidance?